ui_src/src/domain/settings/profile/imgLoader/index.js

```diff
@@ -1,149 +0,0 @@
-// Copyright 2021-2022 The Memphis Authors
```

Comment:
Why you delete this file and did not use it?

Reply:
Done